project/src/test/java/com/google/sps/EventDetailTest.java

```diff
@@ -26,8 +28,10 @@
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
 import org.mockito.Mockito;
+import org.skyscreamer.jsonassert.JSONAssert;
 import org.springframework.mock.web.MockServletContext;
 
+
 @RunWith(JUnit4.class)
 public final class EventDetailTest {
```

Comment:
Also test the case when the servlet returns SC_NOT_FOUND

Reply:
done